### ISSUE TITLE
support for driver 415(.xx)?

### ISSUE THREAD

**Issue by hasezoey** [keylase/nvidia-patch#12]
Linux (Mint)
GTX 1080
Drivers from launchpad graphics-drivers

**Comment by keylase** [comment]
Can you test it and write about results?

**Comment by hasezoey** [comment]
415.xx or more specific 415.18

**Comment by Snawoot** [comment]
@hasezoey Please checkout branch [upstream_series415](https://github.com/keylase/nvidia-patch/tree/upstream_series415), test it and let us know if it works.

**Comment by Snawoot** [comment]
@keylase Got to build test environment and re-run GDB analysis. I hope I'll make some progress within a week.

**Comment by keylase** [comment]
I assume that the code base has changed a bit, and a new byte-sequence is needed for replacement. I'll try it too.

**Comment by Snawoot** [comment]
Branch `upstream_series415` has been updated and now is available for testing.

**Comment by edasque** [comment]
415.25 is the latest though

**Comment by edasque** [comment]
@Snawoot this is what I got:
```
➜  nvidia-patch git:(upstream_series415) ✗ sudo bash ./patch.sh
[sudo] password for ed:                                                                                                         
./patch.sh: line 52: syntax error near unexpected token `<<<'                                                                   
./patch.sh: line 52: `<<<<<<< HEAD'                                                                                             
./patch.sh: line 53: =======: command not found                                                                                 
./patch.sh: line 54: [415.25]=s/\x00\x00\x00\x84\xc0\x0f\x84\x40\xfd\xff\xff/\x00\x00\x00\x84\xc0\x90\x90\x90\x90\x90\x90/g: No such file or directory                                                                                                          
./patch.sh: line 55: syntax error near unexpected token `>>'                                                                    
./patch.sh: line 55: `>>>>>>> f30528f248e4ed9162ee682b316658b7c84f2a35'                                                         
➜  nvidia-patch git:(upstream_series415) ✗ 
```

**Comment by edasque** [comment]
I did get this:
```
➜  nvidia-patch git:(upstream_series415) ✗ sudo bash ./patch.sh
[sudo] password for ed:                                                                                                         
./patch.sh: line 52: syntax error near unexpected token `<<<'                                                                   
./patch.sh: line 52: `<<<<<<< HEAD'                                                                                             
./patch.sh: line 53: =======: command not found                                                                                 
./patch.sh: line 54: [415.25]=s/\x00\x00\x00\x84\xc0\x0f\x84\x40\xfd\xff\xff/\x00\x00\x00\x84\xc0\x90\x90\x90\x90\x90\x90/g: No such file or directory                                                                                                          
./patch.sh: line 55: syntax error near unexpected token `>>'                                                                    
./patch.sh: line 55: `>>>>>>> f30528f248e4ed9162ee682b316658b7c84f2a35'                                                         
➜  nvidia-patch git:(upstream_series415) ✗ 
```

**Comment by Snawoot** [comment]
@edasque looks like you had unresolved merge conflicts while you pulled updates from branch

Since I had to rebase this branch, no fast-forward rewind possible. Please force-update your branch:
```bash
git fetch --all && git reset --hard origin/upstream_series415
```

Or just re-checkout repo and this branch from scratch.

**Comment by edasque** [comment]
Seems to have worked, rebooting now and will try to confirm I can run more than 3 processes today

**Comment by niXta1** [comment]
@edasque, any updates?

**Comment by edasque** [comment]
I think it worked!

**Comment by Snawoot** [comment]
Resolved by PR #22
